.github/workflows/pytest.tmpl

```diff
@@ -18,15 +18,26 @@ jobs:
     name: Python ${{ matrix.python-version }} ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/conda_pkgs_dir
+            ~/.cache/pypoetry
+            ~/miniconda*/envs/
+          key: ${{ runner.os }}-${{ matrix.python-version }}-env-${{ hashFiles('ci/environment.yml') }}-${{ hashFiles('poetry.lock') }}
```

Comment:
I think this key needs to updated if `environment.yml` and `poetry.lock` files aren't present